index.rst

```diff
@@ -11,14 +11,78 @@ Bharat-Sim's documentation!
    :caption: Contents:
 
 
-
-Indices and tables
-==================
-
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
 
-How does this lool?
+
+
+
+SIR Model with Agents
+=====================
+
+The Bharat-sim framework allows you to build agent-based models. Agents, are essentially the main component of the simulation. They are entities which are defined with a bunch of parameters and certain rules which they have to obey as the simulation runs.
+
+In the purview of the SIR Model, the agents are people, which go about their daily lives and follow regular schedules. Here, the parameters defining an agent are a unique agent id, age, their infection status, the time for which they remain infected and so on. The simulation allows you to introduce new parameters as and when required.
+
+The rules governing the behaviour of all agents include the particular schedules which they follow and, the functions which allow the agents to transit from one compartment to the other.
```

Comment:
No comma after "and". Furthermore, the Schedule and Behaviours are fine, but the `checkForInfection` function has nothing to do with the framework or the model, it's just one specific behaviour. Use it as an example of a behaviour, don't write about it in this line.